app/javascript/packs/application.scss

```diff
@@ -9,4 +9,4 @@ layout file, like app/views/layouts/application.html.erb
 */
 
 // Adding a little classess CSS library as a temporary look and feel to verify webpack and bundling is working correctly.
```

Comment:
You can probably remove this comment too. Godspeed 🖖

Reply:
Thanks @harimohanraj89 ... it's nice to see you in here :)